ngrok-java-native/src/lib.rs

```
@@ -69,9 +69,12 @@ impl<'local> com_ngrok::RuntimeRs<'local> for RuntimeRsImpl<'local> {
                     .expect("cannot get logger ref");
                 LOGGER.get_or_init(|| logref);
 
+                let log_lvl: Level =
+                    Level::from_str(&logger.get_level(self.env)).unwrap_or(Level::TRACE);
```

`unwrap_or` here is hiding the error giving you the worst possible level 🤔